src/main/plugins/org.dita.xhtml/xsl/xslhtml/troubleshootingdisplay.xsl

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- This file is part of the DITA Open Toolkit project hosted on 
+     Sourceforge.net. See the accompanying license.txt file for 
+     applicable licenses.-->
+<!-- 20170503 SCH: Add support for troubleshooting elements. -->
+
+<xsl:stylesheet version="2.0"
+     xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+     xmlns:related-links="http://dita-ot.sourceforge.net/ns/200709/related-links"
+     xmlns:dita2html="http://dita-ot.sourceforge.net/ns/200801/dita2html"
+     xmlns:ditamsg="http://dita-ot.sourceforge.net/ns/200704/ditamsg"
+     xmlns:xs="http://www.w3.org/2001/XMLSchema"
+     exclude-result-prefixes="related-links dita2html ditamsg xs">
+
+<!-- Determines whether to generate titles for task sections. Values are YES and NO. -->
+<xsl:param name="GENERATE-TASK-LABELS" select="'NO'"/>
+
+<!-- == TASK UNIQUE SUBSTRUCTURES == -->
+
+  <xsl:template match="*[contains(@class,' troubleshooting/troublebody ')]" name="topic.troubleshooting">
```

Comment:
I haven't double checked to be sure, but I don't think this does anything that the core rule for `<body>` doesn't do already - if that's the case, we shouldn't override it, and should use the fallback behavior.